### PR TITLE
add check for NULL conv

### PR DESCRIPTION
**Pull request by shankerwangmiao**
quick bug fix for #87

### CODE REVIEW

File: pam_yubico.c [hunk 1]
```diff
@@ -410,21 +410,25 @@ display_error(pam_handle_t *pamh, const char *message) {
   pmsg[0] = &msg[0];
   msg[0].msg = (char *) message; /* on some systems, pam_message.msg isn't const */
   msg[0].msg_style = PAM_ERROR_MSG;
-  retval = conv->conv(1, pmsg, &resp, conv->appdata_ptr);
-
-  if (retval != PAM_SUCCESS) {
-    D(("conv returned error: %s", pam_strerror (pamh, retval)));
+  if(conv && conv->conv){
```
Comment (klali) [inline pam_yubico.c]: Wouldn't it be cleaner to add this check (is checking conv->conv necessary?) before setting msg[0], just after checking if the pam_get_item() call was ok?

Comment (shankerwangmiao) [inline pam_yubico.c]: yeah, you are totally right. 

checking of conv->conv is just for insurance.

Comment (klali) [inline pam_yubico.c]: Do you want to update the pull request to that (check before msg and return early) or should I just commit it?

Comment (shankerwangmiao) [inline pam_yubico.c]: yeah, i'll update

**Comment by klali** [comment]
Great, merging this.

**Pull request merged**